dbt/parser.py

```diff
@@ -149,6 +149,8 @@ def parse_node(node, node_path, root_project_config, package_project_config,
 
     context['ref'] = __ref(parsed_node)
     context['config'] = __config(parsed_node, config)
+    context['var'] = lambda x: ''
```

Comment:
can we do `context['var'] = lambda *args: ''`

`var()` can optionally take a second arg, the default value if none is provided